auth_saml/models/auth_saml.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (C) 2010-2016 XCG Consulting <http://odoo.consulting>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+import json as simplejson
```

Comment:
Why? Just use it as `json` to avoid confusion:

```suggestion
import json
```